data/net/minecraft/world/entity/ai/attributes/AttributeMap.mapping

```diff
@@ -17,6 +19,20 @@ CLASS net/minecraft/world/entity/ai/attributes/AttributeMap
 	METHOD hasModifier (Lnet/minecraft/world/entity/ai/attributes/Attribute;Ljava/util/UUID;)Z
 		ARG 1 attribute
 		ARG 2 uuid
+	METHOD lambda$addTransientAttributeModifiers$3 (Lnet/minecraft/world/entity/ai/attributes/Attribute;Lnet/minecraft/world/entity/ai/attributes/AttributeModifier;)V
+		ARG 1 attribute
+		ARG 2 modifier
+	METHOD lambda$assignValues$4 (Lnet/minecraft/world/entity/ai/attributes/AttributeInstance;)V
+		ARG 1 instance
+	METHOD lambda$getInstance$1 (Lnet/minecraft/world/entity/ai/attributes/Attribute;)Lnet/minecraft/world/entity/ai/attributes/AttributeInstance;
+		ARG 1 attribute
```

Comment:
This seems to be a duplicate of the method parameter? Not sure how it's getting past the validate task.